### DISCUSSION TITLE
have two spelling problems in nc official site about lua example

### DISCUSSION THREAD

**Discussion by spaidermfg** [NvChad/NvChad#1837] [Q&A]
[address](https://nvchad.com/docs/quickstart/learn-lua)

<img width="350" alt="截屏2023-03-18 00 17 36" src="https://user-images.githubusercontent.com/46310887/225962405-677ae0e9-b12b-45be-903a-dc1e5dc5bf5a.png">
<img width="453" alt="截屏2023-03-18 00 17 53" src="https://user-images.githubusercontent.com/46310887/225962472-580b15a7-515b-47c8-8126-692fb280a715.png">

**Comment by siduck** [comment]
hi @spaidermfg what are the problems? looks fine to me

**Reply by spaidermfg** [reply]
hi, Is print misspelled as pprint?

**Reply by siduck** [reply]
oh yea, i just looked now. Will fix when our site is back. its currently down

**Reply by spaidermfg** [reply]
ok, thanks!
